wordpress-mobile/WordPress-iOS
# [Notifications P1] Update spacings around 3-avatar row & section headers

## Description
Improves the spacings around 3-avatar row & section headers after a call with @osullivanchris .

We have reverted/removed the `HeaderPosition` @justtwago . Keeping the same vertical padding for all section headers looked better. Let me/us know if you have concerns about it.

## Testing Steps
1. Install & Login Jetpack App
2. Navigate to Notifications Tab
3. ✅ Verify the vertical spacing around Section Headers: "Today", "Yesterday" etc.
4. ✅ Verify the vertical spacing around each type of avatar combination: Single, double, triple.
5. ✅ Repeat step 4 with larger dynamic fonts.

## Regression Notes
1. Potential unintended areas of impact
N/A

6. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A

7. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:


Testing checklist:
